.github/workflows/ci.yml

```diff
@@ -106,24 +106,24 @@ jobs:
       - name: nextest partition ${{ matrix.partition }}/8
         run: cargo nextest run --no-fail-fast --partition 'count:${{ matrix.partition }}/8' --archive-file 'nextest-archive-${{ matrix.os }}.tar.zst' e2e
 
-  test-scarb-2-8-3:
-    name: Test scarb 2.8.3
+  test-scarb-2-8-5:
```

Comment:
Do we even need this step? If all of the tests will run this version anyway?

Reply:
If these are meant to only run on Ubuntu (not on Windows) and not a specific Scarb version, let's rename these accordingly.

Reply:
In fact we can entirely remove it.